lockbox-ios/Presenter/SettingListPresenter.swift

```diff
@@ -79,7 +93,7 @@ class SettingListPresenter {
 }
 
 extension SettingListPresenter {
-    fileprivate func settingsWithBiometricLoginEnabled(_ enabled: Bool, autoLock: AutoLockSetting?, preferredBrowser: PreferredBrowserSetting) -> [SettingSectionModel] { // swiftlint:disable:this line_length
+    fileprivate func settingsWithBiometricLoginEnabled(_ enabled: Bool, autoLock: AutoLockSetting?, preferredBrowser: PreferredBrowserSetting, usageDataEnabled: Bool) -> [SettingSectionModel] { // swiftlint:disable:this line_length
```

Comment:
Can we break this function signature into multiple lines? It's getting a little unwieldy.